centreon-ha/ocf-scripts/mariadb-centreon-common.sh

```
@@ -318,3 +336,92 @@ mysql_common_stop()
+
+        echo -ne "[$(date)][\e[33mWARNING\e[0m]: ${message}\n" >> ${OCF_RESKEY_repli_log_file}
+    fi
+}
+
+# $1: the message that'il be logged
+repli_log_error()
+{
+    if [ ${OCF_RESKEY_repli_log_level} -ge 1 ];then
+        message="$1"
+
+        echo -ne "[$(date)][\e[31mERROR\e[0m]: ${message}\n" >> ${OCF_RESKEY_repli_log_file}
```

Shellcheck (0.10.0)

[warning] In POSIX sh, echo flags are undefined.

(SC3037)